tests/test-cases/demo-custom-command.yaml

```
@@ -0,0 +1,29 @@
+tests:
+  - name: atmos greet with args
+    enabled: true
+    description: "Validate atmos custom command greet runs with argument provided."
+    workdir: "../examples/demo-custom-command/"
+    command: "atmos"
+    args:
+      - "greet"
+      - "Andrey"
+    expect:
+      stdout:
+        - "Hello, Andrey\n"
```

Let's use Neo from the Matrix

```suggestion
      - "Neo"
    expect:
      stdout:
        - "Hello, Neo\n"
```